tests/Sniffs/TypeHints/data/disallowArrayTypeHintSyntaxErrors.fixed.php

```diff
@@ -42,4 +42,20 @@ class Whatever
 	/** @var array<int>|string */
 	public $k;
 
+	/** @return iterable<array<string>> */
```

Comment:
Followed https://github.com/slevomat/coding-standard/blob/fdab228c1800db9e65a62dde306bfa856d98fcc3/tests/Sniffs/TypeHints/data/disallowArrayTypeHintSyntaxErrors.fixed.php#L30 in keeping nested items as `array`.